contrib/runners/windows_runner/windows_runner/windows_command_runner.py

```diff
@@ -93,13 +93,12 @@ def run(self, action_parameters):
         if exit_code != 0:
             error = self._parse_winexe_error(stdout=stdout, stderr=stderr)
 
-        result = stdout
-
         output = {
             'stdout': stdout,
             'stderr': stderr,
-            'exit_code': exit_code,
-            'result': result
+            'return_code': exit_code,
```

Comment:
I personally prefer exit code over return code, but I made the change for the consistency reasons.

Reply:
Agreed - I think that consistency here is more helpful